src/hyperfine/benchmark.rs

```diff
@@ -207,11 +214,14 @@ pub fn run_benchmark(
 
     // Warmup phase
     if options.warmup_count > 0 {
-        let progress_bar = get_progress_bar(
-            options.warmup_count,
-            "Performing warmup runs",
-            options.output_style,
-        );
+        let mut progress_bar: Option<ProgressBar> = None;
+        if options.output_style != OutputStyleOption::None { 
+                progress_bar = Some(get_progress_bar(
+                options.warmup_count,
+                "Performing warmup runs",
+                options.output_style,
+            ));
+        }
```

Comment:
We could avoid `mut` here by writing this as
```
let progress_bar = if … {
  …
} else {
  …
};
```

Reply:
thanks for great suggestion!